app/pulp/app/pulp_manager.py

```diff
@@ -0,0 +1,14 @@
+"""This module provides system wide command to interface with the pulp platform.
+
+In future commands that can be issued should be limitted this will probably be achieved
+by implementing own ManagmentUtility similar to this one:
+https://github.com/django/django/blob/stable/1.8.x/django/core/management/__init__.py
+"""
+import os
+import sys
+
+
+def pulp_manager_entry_point():
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pulp.app.settings")
```

Comment:
We don't want setdefault here, since this is `pulp-manager`, not `generic-django-manager`. The story says "have all commands be passed along to the manage.py for pulp_app", which I take to mean that we need this to always use the pulp.app settings regardless of what `DJANGO_SETTINGS_MODULE` is set to.